src/set_theory/ordinal/basic.lean

```diff
@@ -746,8 +746,17 @@ instance : has_zero ordinal :=
 
 instance : inhabited ordinal := ⟨0⟩
 
-theorem zero_eq_type_empty : 0 = @type empty empty_relation _ :=
```

Comment:
This one is redundant because it's just `type_eq_zero_of_empty empty_relation`, right?

Reply:
Yes, and written backwards no less.